conformance/README.md

```diff
@@ -561,14 +601,16 @@ type.
 These rules describe the overall shape of test cases:
 
 ```ebnf
-test ::=  "("  "ion_1_0"  name-string?  fragment*  continuation  ")"
+test ::=  "("  "document" name-string?             continuation  ")"
```

Comment:
I have not intended to express that `document` cannot work with fragments in its grammar rule.  [What I said](https://github.com/amazon-ion/ion-tests/pull/98#discussion_r1628563605) was that doing so leads to more semantics to define here, and more for each implementation to build and test.  The benefit is slightly less syntax for test-cases that use `document`, but I don't think there's any reason to use `document` except for a small handful of cases to test that a no-IVM document behaves the same as if `$ion_1_0` were present. 

For the most part, this entire grammar clause is pedagogical, to enable a crisp and precise definition of the other three root forms.  It also enables the changes to `ion_1_0` such that it does 50% less work than before, and `ion_1_x` does 1/3 less work, because they don't need to cover the no-IVM case through the entire test suite.

I aim to design languages from as few primitives as possible, because that's what must be most carefully specified, and represent the primary parts to build.  The definitions here are such that
* there is a single "bootstrap" form
* that form does one thing (create an empty document)
* all of the semantics of fragment assembly is delegated to (ie reused from) the definition of a continuation.

In other words, this design is very DRY, at the cost of more verbiage for one or two test files total.

Reply:
After building this change I came to the conclusion that the implementation delta was near trivial to allow more flexibility.  See https://github.com/amazon-ion/ion-tests/pull/108 for changes that I think align with your thinking.